src/Ui.GraphiQL/Internal/graphiql.cshtml

```diff
@@ -8,148 +8,175 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="UTF-8">
-	<style>
-		body {
-			height: 100%;
-			margin: 0;
-			width: 100%;
-			overflow: hidden;
-		}
+  <meta charset="UTF-8">
+  <style>
+    body {
+      height: 100%;
+      margin: 0;
+      width: 100%;
+      overflow: hidden;
+    }
 
-		#graphiql {
-			height: 100vh;
-		}
-	</style>
+    #graphiql {
+      height: 100vh;
+    }
+  </style>
 
-	<!--
-	  This GraphiQL example depends on Promise and fetch, which are available in
-	  modern browsers, but can be "polyfilled" for older browsers.
-	  GraphiQL itself depends on React DOM.
-	  If you do not want to rely on a CDN, you can host these files locally or
-	  include them directly in your favored resource bunder.
-	-->
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/es6-promise/4.1.1/es6-promise.auto.min.js"></script>
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.min.js"></script>
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/react/16.2.0/umd/react.production.min.js"></script>
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.2.0/umd/react-dom.production.min.js"></script>
+  <!--
+    This GraphiQL example depends on Promise and fetch, which are available in
+    modern browsers, but can be "polyfilled" for older browsers.
+    GraphiQL itself depends on React DOM.
+    If you do not want to rely on a CDN, you can host these files locally or
+    include them directly in your favored resource bunder.
+  -->
+  <script
+    src="https://cdn.jsdelivr.net/npm/whatwg-fetch@2.0.3/fetch.min.js"
+    integrity="sha384-dcF7KoWRaRpjcNbVPUFgatYgAijf8DqW6NWuqLdfB5Sb4Cdbb8iHX7bHsl9YhpKa"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://unpkg.com/react@16.14.0/umd/react.production.min.js"
```

Comment:
Why do you use SRI in some script/link tags and in some not ?

Reply:
Most of the code has been retrieved from the CDN example file of graphiql repository: https://github.com/graphql/graphiql/blob/main/examples/graphiql-cdn/index.html and actually there are no SRI in there. And for the other dependencies, I added it whenever possible. I could try to add them everywhere,in this pull request or another. What do you think?

Reply:
Please add them here. https://www.srihash.org/ helps a lot.

Reply:
And done! No issue while testing it. Great tool you provided, thanks.